static/js/vue-cdr-access/src/components/collectionBrowseWrapper.vue

```diff
@@ -14,20 +14,28 @@ Top level component used for the collection browse page
             </div>
         </div>
     </div>
+    <not-found v-if="show_404" :display-header="false"></not-found>
```

Comment:
You may have already looked at this, but I was wondering if it'd make sense to have the not-found and not-available modules up at the App.vue root level rather than in each submodule? Sort of like this example:
https://stackoverflow.com/a/60524363

Reply:
It seems like we'd have to move the data retrieval queries up into the router since we don't necessarily know if there's an issue until we try to fetch the data.